qiita_db/util.py

```diff
@@ -1950,6 +1952,10 @@ def open_file(filepath_or, *args, **kwargs):
             fh.close()
 
 
+def artifact_visibilities_to_skip():
+    return tuple([qdb.util.convert_to_id('archived', "visibility")])
```

Comment:
A new function for one line of code seems a little excessive. However I appreciate the readability. Not sure I would suggest unfolding it.

Reply:
I tried adding as a global variable in the artifact.py and as a member of the Artifact object but turns out that this is not possible because the code will try to initialize those variables when the code is initiated (and there is no database) so it fails, [for example](https://github.com/qiita-spots/qiita/runs/6801800785?check_suite_focus=true).